packages/sync/src/connect-webrtc.js

```diff
@@ -19,7 +23,12 @@ import { WebrtcProvider } from 'y-webrtc';
  */
 export function connectWebRTC( objectId, objectType, doc ) {
 	const docName = `${ objectType }-${ objectId }`;
-	new WebrtcProvider( docName, doc, {
+	new WebrtcProviderWithHttpSignaling( docName, doc, {
+		signaling: [
+			//'ws://localhost:4444',
+			// @ts-ignore
+			window.wp.ajax.settings.url,
```

Comment:
We should transform `connectWebRTC` function to a `createWebRTCConnection` function (currying) where the first argument is a config and one of the config options is the URL of the signaling server.

In other words, the `sync` package should be independent.